compiler/dyno/include/chpl/resolution/resolution-types.h

```diff
@@ -37,6 +37,10 @@
 namespace chpl {
 namespace resolution {
 
+typedef enum {
+  IGNORE, USE, FOR_OTHER_FIELDS
```

Comment:
Could we have a docs comment above each of these explaining its meaning? Also I would find `IGNORE_DEFAULTS`, `USE_DEFAULTS`, `USE_DEFAULT_OTHER_FIELDS` more intuitive.